### ISSUE TITLE
Camera unavailable + code showing 

### ISSUE THREAD

**Issue by nibagit** [dermotduffy/frigate-hass-card#1439]
frigate 0.13.2-6476F8A
frigate Lovelace card v5.2.0

Since v13 and v5.2.0, I am having Camera unavailable + code (camera config) showing instead of "no frames received, etc.." (Frigate error) as in previous versions when the camera is not being picked up by Frigate (off or unavailable on network).  

Now this might be more of a feature request, however it would be awesome if you can have options how to deal with cameras that are unavailable, being: 
- static picture
- last image + with custom text in the middle 
- leave cameras that are unavailable out of the line up     

Basically, I tend to switch off certain indoor cameras based upon the status of home (armed, disarmed). 

Anyways thanks for the awesome work on this card!!!

**Comment by GiowGiow** [comment]
I'm having the same issue. Presumably because I use wi-fi cameras, so they are not as stable as wired ones.

**Comment by Lockie85** [comment]
I'm having an issue with this to. I have a Frigate card with multiple cameras on it and if one camera has connection issues the entire card and all its cameras stop working. Feels like this shouldn't be the case.

**Comment by DReffects** [comment]
same issue here. upgarded from working 0.13 to 0.14 beta, cards stopped working

**Comment by dermotduffy** [comment]
This is already fixed to some degree on the `dev` branch, so feel free to test that out (no support offered, though): https://card.camera/#/advanced-installation?id=unreleased-versions .

I like the idea of having a configurable option of how this is treated as described in the original post on this issue, although some of those are easier than others (e.g. having a local copy of the last usable frame might be great, but would involve saving every frame which would have a performance impact -- but others are more doable).

**Comment by dermotduffy** [comment]
The new beta build (https://github.com/dermotduffy/frigate-hass-card/releases/tag/v6.0.0-beta.2) will not block the whole card if a single camera goes unavailable. Please let me know if that is sufficient, or if you think the most advanced ideas described at the start are still worth it.
